examples/sdp3x_default_address/sdp3x_default_address.ino

```diff
@@ -9,17 +9,18 @@ void setup() {
   Wire.begin();
   Serial.begin(9600);
   delay(1000); // let serial console settle
-
-  int ret = sdp.init();
-  if (ret == 0) {
+  bool error = false;
+  do{
+    int ret = sdp.init();
+    if (ret == 0) {
       Serial.print("init(): success\n");
-  } else {
+    } else {
+	  error = true;
       Serial.print("init(): failed, ret = ");
       Serial.println(ret);
-      while (true) {
-        delay(1000);
+      delay(1000);
       }
-  }
+  }while(error);
```

Comment:
space before `while` - also if we drop `error` the condition becomes `while(true)`